Java/tests/src/test/java/com/microsoft/recognizers/text/tests/datetime/DateTimeTest.java

```diff
@@ -41,39 +41,33 @@ protected void recognizeAndAssert(TestCase currentCase) {
         List<ModelResult> results = recognize(currentCase);
 
         // assert
-        assertResultsDateTime(currentCase, results);
+        assertResults(currentCase, results, Collections.emptyList());
     }
 
-    public static <T extends ModelResult> void assertResultsDateTime(TestCase currentCase, List<T> results) {
-
-        List<ExtendedModelResult> expectedResults = readExpectedResults(ExtendedModelResult.class, currentCase.results);
-        Assert.assertEquals(getMessage(currentCase, "\"Result Count\""), expectedResults.size(), results.size());
-
-        IntStream.range(0, expectedResults.size())
-                .mapToObj(i -> Pair.with(expectedResults.get(i), results.get(i)))
-                .forEach(t -> {
-                    ExtendedModelResult expected = t.getValue0();
-                    T actual = t.getValue1();
-
-                    Assert.assertEquals(getMessage(currentCase, "typeName"), expected.typeName, actual.typeName);
-                    Assert.assertEquals(getMessage(currentCase, "text"), expected.text, actual.text);
-                    if (actual instanceof ExtendedModelResult) {
-                        Assert.assertEquals(getMessage(currentCase, "parentText"),
-                                            expected.parentText, ((ExtendedModelResult)actual).parentText);
-                    }
+    @Override
+    protected void assertModel(ModelResult expected,
+                               ModelResult actual,
+                               TestCase currentCase,
+                               List<String> testResolutionKeys) {
```

Comment:
Having all these in one line it makes the checklist to complain, right?

Reply:
yes